README.md

```diff
@@ -522,37 +563,60 @@
     if (test)
       return false;
 
+    // acceptable (only for very simple statements and early function return)
```

Comment:
I'm not at all a fan of this at all, and would change it if I saw it in any codebase I was in, but, I'm not going to fight against it here if it's super important to you.